integration-tests/tests/src/tests/sync/mongo-db-client.ts

```diff
@@ -93,6 +93,438 @@ describe.skipIf(environment.missingServer, "MongoDB Client", function () {
   });
 
   describe("MongoDBCollection", function () {
+    const insertedId1 = 1;
```

Comment:
Is there a benefit of having `insertedId1 = 1; insertedId2 = 2;` instead of just using the `1, 2...`?

Reply:
If we have them as variables rather than hardcoded in each test I think it makes the tests more readable since it would hint we're dealing with an id that has already been inserted. Same for the `nonExistentId` variable. Also, it's a lot easier to change the values if ever needed.